Chef Vault incompatible with old-style client cert

sean-horn opened the pull request:
Chef Vault does not work with old-style chef-client certs, only with current public/private keypairs

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

jamescott commented:
@delivery approve

chef-delivery commented:
Change: d57c4b7e-5812-4d74-b77b-ede57bad78c6 approved by: @jamescott

chef-delivery commented:
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

sean-horn committed:
Chef Vault incompatible with old-style client cert

Chef Vault does not work with old-style chef-client certs, only with current public/private keypairs

irvingpop commented:
Questions:   
- How old of a chef client would you need (era or major version) to get those?  
- How would a customer know that those keys exist in their environment?  (is there any way to proactively audit)

Commit:
Copyedits warning

kagarmoe commented:
Do we have a documented workflow for updating an older system to use chef vault?  If I'm running a 2 year-old system (or older?) and throw in ``gem chef-vault``, how does it go for me?

kagarmoe commented:
![tenor-195759238](https://user-images.githubusercontent.com/4400151/27460227-5f959dac-5767-11e7-81ae-a510e6bbdfd2.gif)
